packages/strapi-plugin-i18n/config/functions/migrations/field/migrateForMongoose.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const { shouldBeProcesseed, getUpdatesInfo } = require('./utils');
+
+const BATCH_SIZE = 1000;
+
+const migrateForMongoose = async ({ model, attributesToMigrate, locales }) => {
+  const processedLocaleCodes = [];
+  for (const locale of locales) {
+    let batchCount = BATCH_SIZE;
+    let lastId;
+    while (batchCount === BATCH_SIZE) {
+      const findParams = { locale: locale.code };
+      if (lastId) {
+        findParams._id = { $gt: lastId };
+      }
+
+      const batch = await model
+        .find(findParams, [...attributesToMigrate, 'locale', 'localizations'])
+        .sort({ _id: 1 })
+        .limit(BATCH_SIZE);
+
+      if (batch.length > 0) {
+        lastId = batch[batch.length - 1]._id;
+      }
+      batchCount = batch.length;
+
+      const entriesToProcess = batch.filter(shouldBeProcesseed);
```

Comment:
that doesn't seem to work. shouldBeprocessed needs to be called with the locales apparently